### PR TITLE
Update meeting-policies-in-teams.md

### PR DESCRIPTION
**Pull request by RuiPereiraTabares**
confirmed with Andrii Matukhno

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/meeting-policies-in-teams.md [hunk 1]
```diff
@@ -37,7 +37,8 @@ You can implement policies in the following ways, which affect the meeting exper
 |Per-organizer and per-user     |When you implement a combination of a per-organizer and per-user policy, certain features are restricted for meeting participants based on their policy and the organizer's policy. For example, **Allow cloud recording** is a per-organizer and per-user policy. Turn on this setting to allow the meeting organizer and participants to start and stop a recording.
 
 By default, a policy named Global (Org-wide default) is created. All users in your organization are assigned the Global meeting policy by default. You can either make changes to it or create one or more custom policies and assign users to them. Users will get the Global policy unless you create and assign a custom policy. When you create a custom policy, you can allow or prevent certain features from being available to your users, and then assign it to one or more users who will have the settings applied to them.
-
+-Note-
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:40]: > [!NOTE]

File: Teams/meeting-policies-in-teams.md [hunk 2]
```diff
@@ -37,7 +37,8 @@ You can implement policies in the following ways, which affect the meeting exper
...
+Meeting details buttom will be aviliable if a user has the audioconference licenses enabled or the user is allow for audioconferencing, if not, the meeting details will not be aviliable
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-in-teams.md:41]:
```suggestion
> The **Meeting Details** button will be available if the user has an Audio Conferencing license enabled or if the user is allowed for Audio Conferencing. If not, the meeting details will not be available.
```

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**